app/src/main/res/drawable/search_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:padding="12dp"
+    android:shape="rectangle" >
+    <solid android:color="#FFFFFF" />
```

Comment:
instead of contant color here, define new color in https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/res/values/colors.xml
and use that here